docs/doc.md

```diff
@@ -61,7 +61,7 @@ Where:
 
 -   `shard_id` is the shard ID of the shard;
 -   `expected_period_number` is the period number in which this collation expects to be included; this is calculated as `period_number = floor(block.number / PERIOD_LENGTH)`;
--   `period_start_prevhash` is the block hash of block `PERIOD_LENGTH * expected_period_number - 1` (i.e., it is the hash of the last block before the expected period starts). Opcodes in the shard that refer to block data (e.g. NUMBER and DIFFICULTY) will refer to the data of this block, with the exception of COINBASE, which will refer to the shard coinbase;
+-   `period_start_prevhash` is the block hash of the last block `PERIOD_LENGTH * expected_period_number - 1` (i.e., it is the hash of the last block before the expected period starts). Opcodes in the shard that refer to block data (e.g. NUMBER and DIFFICULTY) will refer to the data of this block, with the exception of COINBASE, which will refer to the shard coinbase;
```

Comment:
IMO the original one seems more correct.
e.g. `expected_period_number = 1`, then `PERIOD_LENGTH * expected_period_number - 1 = 4`, and therefore `period_start_prevhash` is the block hash of the block `4`.

Reply:
I'm going to revert back to what it was and agree with you on this one